api/metadata_handler_medium_test.go

```diff
@@ -0,0 +1,133 @@
+// +build medium
+
+// Copyright 2018 The WPT Dashboard Project. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package api
+
+import (
+	"testing"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/web-platform-tests/wpt.fyi/api/query"
+	"github.com/web-platform-tests/wpt.fyi/shared"
+)
+
+
+func TestFilterMetadataHanlder_Success(t *testing.T) {
+	handler := func(w http.ResponseWriter, r *http.Request) {
+		http.ServeFile(w, r, "../shared/metadata_testdata/gzip_testfile.tar.gz")
+	}
+	server := httptest.NewServer(http.HandlerFunc(handler))
+	defer server.Close()
+
+	r := httptest.NewRequest("GET", "/abd/api/metadata?product=chrome&product=safari", nil)
+	w := httptest.NewRecorder()
+	client := &http.Client{}
+
+	metadataHandler := MetadataHandler{nil, client, server.URL}
+	metadataHandler.ServeHTTP(w, r)
+
+	assert.Equal(t, http.StatusOK, w.Code)
+	res := w.Body.String()
+	assert.Equal(t, "[{\"test\":\"/IndexedDB/bindings-inject-key.html\",\"urls\":[\"bugs.chromium.org/p/chromium/issues/detail?id=934844\",\"\"]},{\"test\":\"/html/browsers/history/the-history-interface/007.html\",\"urls\":[\"bugs.chromium.org/p/chromium/issues/detail?id=592874\",\"\"]}]", res)
```

Comment:
Use backticks (``) instead of double quotes to avoid having to escape all the double quotes in the JSON string.